app/assets/stylesheets/active_admin/components/_tables.scss

```diff
@@ -5,6 +5,9 @@ table tr {
   td {
     vertical-align: top;
   }
+  td.numeric {
```

Comment:
Merge rule `td.numeric` with rule on line 5<br>Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3